nfc_ros/README.md

```diff
@@ -0,0 +1,27 @@
+# nfc_ros
+
+This package is for nfc communication.
+
+## How to use
+
+Before use this, please install nfcpy and setup udev for your device.
+
+```bash
+pip3 install nfcpy
+```
+
+And then, connect your NFCReader to PC and run
+
+```bash
+python3 -m nfc
+```
+
+This will show messages and if you need to do somethin, there will be an instruction. please follow that.
+
+After that, please build this package and run.
+
+```bash
+roslaunch nfc_ros demo.launch
+```
+
```

Comment:
please add comment what we can get from this program. whics topics should be subscribed, what did it inform you.

Reply:
Added.